gce-scripts/combine_shards.py

```diff
@@ -0,0 +1,29 @@
+"""Combines pshtt shards into one final data file."""
+import json
+import sys
+
+
+def main():
+    if (len(sys.argv)) < 2:
+        print('you need a filename!')
+        exit(1)
+    # Master file is the file with the list of filenames to intake.
+    # Fileception.
+    master_file = sys.argv[1]
```

Comment:
May not be, but is it necessary to do any error handling here? Either check the file exists or catch the exception?